services/movie.py

```diff
@@ -0,0 +1,46 @@
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None,
```

Comment:
change `list[int] = None` to `Optional[list[int]] = None` everywhere. If it accepts `None` as default, it must be `Optional`.